src/applications/search/containers/SearchApp.jsx

```diff
@@ -36,21 +36,21 @@ class SearchApp extends React.Component {
   constructor(props) {
     super(props);
 
-    let userInputFromAddress = '';
-    let page;
+    let userInputFromURL = '';
+    let pageFromURL;
 
     if (this.props.router.location.query) {
-      userInputFromAddress = this.props.router.location.query.query;
-      page = this.props.router.location.query.page;
+      userInputFromURL = this.props.router?.location?.query?.query;
+      pageFromURL = this.props.router?.location?.query?.page;
     }
```

Comment:
I don't think you need this if statement since you're doing `?.` now, you could make `userInputFromURL` and `pageFromURL` `const`s if you want! 🙂